focus-areas/business-risk/committers.md

```diff
@@ -51,9 +56,20 @@ ORDER BY
     counter DESC
 ```
 
-This expression allows an end user to filter by commit count thresholds easily, and the number of rows returned is the "Total_Committers" for the repository. 
-
-[Grimoire Lab](https://chaoss.biterg.io/app/kibana#/dashboard/Git) additionally provides insight into committers. 
+This expression allows an end user to filter by commit count thresholds easily, and the number of rows returned is the "Total\_Committers" for the repository.
 
 ## References
-1. Nora McDonald, Kelly Blincoe, Eva Petakovic, and Sean Goggins. 2014. Modeling Distributed Collaboration on GitHub. Advances in Complex Systems 17(7 & 8).
+
+1.  Nora McDonald, Kelly Blincoe, Eva Petakovic, and Sean Goggins. 2014. Modeling Distributed Collaboration on GitHub. Advances in Complex Systems 17(7 & 8).
+2.  [Grimoire Lab](https://chaoss.biterg.io/app/kibana#/dashboard/Git)
+
+## Additional Information
+
+To edit this metric please [submit a Change Request here](https://github.com/chaoss/wg-risk/blob/main/focus-areas/business-risk/committers.md)
+
+To reference this metric in software or publications please use this stable URL: <https://chaoss.community/?p=3945>
+
+<!-- # For groupings in the knowledge base
+Context tags: Contributor, Platform
+Keyword tags: risk, contributors, committers, maintainers
+-→ 

```

Comment:
Since you will be in the file making changes, this last character comes through kind of weird and disrupts the footer on the website. Can you replace that last character with `-->`? Thank you!